code/game/gamemodes/changeling/helpers/_store.dm

```diff
@@ -53,10 +53,17 @@ var/list/datum/power/changeling/powerinstances = list()
 /datum/power/changeling/fakedeath
 	name = "Regenerative Stasis"
 	desc = "We fake our own death, imperceptible to even the best of doctors. We can choose when to re-animate."
-	helptext = "Can be used before or after death. Duration varies greatly."
+	helptext = "Can only be used before death. CAN ONLY BE USED ONCE."
```

Comment:
```suggestion
	helptext = "Can only be used before death. CAN ONLY BE USED ONCE."
```
bold or SPAN_DANGER (or similarly) for the latter part?

Reply:
not sure if that's necessary, capital letters should be fine

Reply:
👍